config/Shared/config_default-middleware-development.php

```diff
@@ -16,3 +16,6 @@
 $config[PropelConstants::ZED_DB_ENGINE] = $config[PropelConstants::ZED_DB_ENGINE_PGSQL];
 
 $config[ProcessConstants::PRODUCT_IMPORT_FILE_PATH] = APPLICATION_ROOT_DIR . '/files/product.txt';
+$config[ProcessConstants::PRODUCT_IMPORT_OUTPUT_PATH] = APPLICATION_ROOT_DIR . '/files/output/product_import.output';
```

Comment:
Maybe adding some comments on what is for or from what might save some confusion here.